.drone.yml

```diff
@@ -0,0 +1,70 @@
+---
+workspace:
+  base: /go
+  path: src/github.com/NYTimes/drone-gae
+
+build_config: &build_config
+  image: golang:1
+  environment:
+    - GO111MODULE=on
+    - CGO_ENABLED=0
+    - GOPATH=/go
+  commands:
+    - go build -a -ldflags "-X main.version=${DRONE_TAG:-other} -X main.rev=${DRONE_COMMIT}"
+
+publish_config: &publish_config
+  image: plugins/docker
+  repo: nytimes/drone-gae
+  secrets:
+    - docker_username
+    - docker_password
+
+pipeline:
+  test:
+    image: golang:1
+    environment:
+      - GO111MODULE=on
+      - CGO_ENABLED=0
+      - GOPATH=/go
+    commands:
+      - go mod download
+      - go vet
+      - go test -cover
+
+  build:
+    <<: *build_config
+    when:
+      branch:
+        - master
+
+  build_release:
+    <<: *build_config
+    when:
+      event: tag
+
+  publish_latest:
+    <<: *publish_config
+    auto_tag: true
+
+  slack:
+    image: plugins/slack
+    channel: dv-notifications
+    webhook: "$$SLACK_WEBHOOK"
```

Comment:
This isn't necessary AFAIK, the secret should set the variable already.

Reply:
And I'm not sure this even works (as in, will it evaluate the env variable to populate that property).

I think Tony is right, you should be fine as long as the env variable with the secret is `SLACK_WEBHOOK` (which it is).

Reply:
I've swapped it out to match the docs here: http://plugins.drone.io/drone-plugins/drone-slack/

these also contradict the other secrets docs so this should be fun